src/query/formats/src/file_format_type.rs

```diff
@@ -27,15 +27,15 @@ use common_meta_types::StageFileFormatType;
 use common_settings::Settings;
 
 use super::clickhouse::ClickhouseSuffix;
+use crate::output_format::output_format_csv::CSVOutputFormat;
```

Comment:
A bit verbose to me, how about `output_format::csv::CSVOutputFormat`?

Since `CSVOutputFormat` is self-contained, we can export as `crate::CSVOutputFormat`.

Reply:
Let's do this later until it's mature. We have lots of crates to publish.

Reply:
thank you, refactor done, please review again.